docs/specification.md

```diff
@@ -152,8 +152,8 @@ type proposal =
   // ^ total amount of votes in favor
   ; downvotes : nat
   // ^ total amount of votes against
-  ; start_date : timestamp
-  // ^ time of submission, used to order proposals
+  ; start_level : blocks
+  // ^ level of submission, used to order proposals
```

Comment:
"level of submission" sounds a bit... wrong? I suggest to use `block level of submission` instead, as in the code comment.